### DISCUSSION TITLE
willing to use rsbuild ?

### DISCUSSION THREAD

**Discussion by honwhy** [doocs/md#370] [General]
check this https://moonvy.com/blog/post/2024/migrate-vite-to-rsbuild/

**Comment by yanglbme** [comment]
Hi @honwhy 

If `rsbuild` offers a better development experience, we would be happy to migrate to `rsbuild`.

**Comment by YangFong** [comment]
从性能上来说，转换之后，速度会有很大提升。

然而我不太喜欢 Rspack，此外，这也就变成了 Webpack 生态，当然，项目现在并没有太多 dev/build plugin 上的依赖，也正是转换的好时候。

我的想法则是：会尝试使用 Rspack，同时，还有 [Farm](https://github.com/farm-fe/farm)。对比表现，若是两者相差不大，则会优先使用后者。后续 Vite@6 若是融入了 Rust，也会考虑转换回来。
